cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -353,6 +348,184 @@
+        {% endif %}
+        {% if controls.categories.filter_category %}
+            <div class="form-l_col
+                        form-l_col-1-3">
+                <div class="form-group">
+                    <label class="form-label-header"
+                           for="categories">
+                        {% if 'leadership-calendar' in controls.categories.page_type %}
+                            Calendars
+                            </label>
+                            {% for slug, name in choices_for_page_type(controls.categories.page_type) %}
+                                {{ _filter_selectable('radio', index, category_label(slug), 'categories_' ~ slug, 'categories', slug) }}
+                            {% endfor %}
+                        {% else %}
+                            Category
+                            </label>
+                            {% for slug, name in choices_for_page_type(controls.categories.page_type) %}
+                                {{ _filter_selectable('checkbox', index, category_label(slug), 'categories_' ~ slug, 'categories', slug) }}
+                            {% endfor %}
+                        {% endif %}
+                </div>
+            </div>
+        {% endif %}
+        {% if controls.topics or controls.authors or controls.date_range %}
+            <div class="form-l_col
+                        form-l_col-2-3">
+                <div class="form-group">
+                    <div class="form-l">
+                        {% if controls.topics %}
+                            <div class="form-l_col
+                                        form-l_col-1-2">
+                                <div class="form-group">
+                                    <label class="form-label-header"
+                                           for="topics">
+                                        Topic
+                                    </label>
+                                    {{ form.render_with_id(form.topics,
+                                       'filter' ~ index ~ '_topics') }}
+                                </div>
+                            </div>
+                        {% endif %}
+                        {% if controls.authors %}
+                            <div class="form-l_col
+                                        form-l_col-1-2">
+                                <div class="form-group">
+                                    <label class="form-label-header"
+                                           for="authors">
+                                        Author
+                                    </label>
+                                    {{ form.render_with_id(form.authors,
+                                       'filter' ~ index ~ '_authors') }}
+                                </div>
+                            </div>
+                        {% endif %}
+                        {% if controls.date_range %}
+                            <div class="form-l_col
```

Comment:
We probably could just use https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/atoms/date-picker.html to render the date picker.